pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -479,7 +479,7 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
         for restriction in restrictions:
             legends = {}
             for legend in restriction['Legend']:
-                type_ = legend['TypeCode']
+                type_ = f"{legend['TypeCode']}_{legend['Geom_Type']}"
```

Comment:
I think, there are no constraints defined on TypeCode from the "Rahmenmodell". Therefore, additional similar issues might occur in the future. Would it not make sense to switch here to the legend entry id?

Reply:
This is correct from the data point of view. Better use the id. It should be available as `identifier` on the LegendEntryRecord

Reply:
@vvmruder should we fix this?
In the JSON I do not see much that I could use as a legend id. Do I need to add it from the DB? OR did I just miss this?

Reply:
Isn't the "SymbolRef" unique for a legend entry?

Reply:
@michmuel OK I just changed it to use `SymbolRef` (its a long key :-) ) 
@voisardf can you check if this is also working for you?